base/services/wlansvc/rpcserver.c

```diff
@@ -171,8 +172,87 @@ DWORD _RpcEnumInterfaces(
 
     return ERROR_SUCCESS;
 #else
-    UNIMPLEMENTED;
-    return ERROR_CALL_NOT_IMPLEMENTED;
+    DWORD dwSize;
+    unsigned int i;
+    MIB_IFROW *pIfRow;
+    MIB_IFTABLE *pIfTable;
+    PWLAN_INTERFACE_INFO_LIST InterfaceList;
+    DWORD len = _countof(pIfRow->bDescr);
+    PWLANSVCHANDLE lpWlanSvcHandle;
+
+    lpWlanSvcHandle = WlanSvcGetHandleEntry(hClientHandle);
+    if (!lpWlanSvcHandle)
+    {
+        return ERROR_INVALID_HANDLE;
+    }
+
+    pIfTable = HeapAlloc(GetProcessHeap(), HEAP_ZERO_MEMORY, sizeof(MIB_IFTABLE));
+    if (!pIfTable)
+    {
+        return ERROR_NOT_ENOUGH_MEMORY;
+    }
+
+    dwSize = sizeof(MIB_IFTABLE);
+    if (GetIfTable(pIfTable, &dwSize, FALSE) == ERROR_INSUFFICIENT_BUFFER)
+    {
+        HeapFree(GetProcessHeap(), 0, pIfTable);
+        pIfTable = HeapAlloc(GetProcessHeap(), HEAP_ZERO_MEMORY, dwSize);
+        if (!pIfTable)
+        {
+            return ERROR_NOT_ENOUGH_MEMORY;
+        }
+    }
+
+    if ((GetIfTable(pIfTable, &dwSize, FALSE)) == NO_ERROR)
+    {
+        dwSize = (pIfTable->dwNumEntries * sizeof(WLAN_INTERFACE_INFO));
+
+        DPRINT1("NumEntries %ld\n",pIfTable->dwNumEntries);
+        __debugbreak();
```

Comment:
Debug leftover?

Reply:
This PR is unfinished and waiting to be picked up by someone else (see the TODO list).
Christoph couldn't figure out the crash at Hackfest, so he committed what he had and hopes for someone to find out why the function still crashes at RPC return.